lib/create.js

```diff
@@ -229,7 +228,7 @@ exports.create = function (project_path, config, options, events) {
         .then(function () {
             return exports.validateProjectName(project_name);
         }).then(function () {
-        // Log the given values for the project
+            // Log the given values for the project
```

Comment:
Generally, I would suggest adding a seperate chore/style PR for these type of changes, but since you are only formatting one line, I guess it is OK ><.

Reply:
OK ;)